specifications/loopback-4-filter-specifications.md

```diff
@@ -0,0 +1,270 @@
+# The LoopBack 4 Filter Specification - Draft 1
+
```

Comment:
I'd like to discuss what's the best place where to keep documents like this one. Using a new top-level folder `specifications` is a fine choice :+1:

Few other alternatives that come to my mind:

- Do we want to ever render the spec on https://loopback.io? Then it maybe better to put the content to `docs/specifications`.
- Since this spec is tied to `@loopback/filter` package, would it make sense to keep it in `packages/filter`? Maybe have a doc page with links to different spec documents we have in our packages, to make them easier to find from a single place.

Also do we need to repeat "loopback" and "specifications" in the file name? Since this file is in the LoopBack 4 repo and inside `specifications` folder, maybe `specifications/filter.md` is sufficient?

Just few ideas to consider. I don't want to bike-shed, feel free to keep what you have now.